### ISSUE TITLE
ERROR: Failed to launch the browser process! spawn 

### ISSUE THREAD

**Issue by vjjv** [drudge/n8n-nodes-puppeteer#2]
Just deployed latest n8n with the one click module to heroku (https://github.com/sarveshpro/n8n-heroku)
Error 503 when installing your community nodes, but marked as installed. Then when I try the node I get this error.

ERROR: Failed to launch the browser process! spawn /data/.n8n/nodes/node_modules/puppeteer/.local-chromium/linux-1002410/chrome-linux/chrome ENOENT TROUBLESHOOTING: https://github.com/puppeteer/puppeteer/blob/main/docs/troubleshooting.md

**Comment by drudge** [comment]
Hi there! Puppeteer has several dependencies that may not be installed by default. This node does not install the dependencies and assumes you have a working version of puppeteer.

You'll need to add those dependencies to the Dockerfile that is referenced in the repository you linked. If you look at the Manual instructions in the README, there are some examples of getting Puppeteer running in Docker:

https://github.com/drudge/n8n-nodes-puppeteer#manual-installation

**Comment by drudge** [comment]
> The same problem. Run.Node:v16.16 in docker

Did you see my comment above? You'll need to add those the puppeteer dependencies to your Docker image for the n8n node to be able to execute Puppeteer. If you've done so, can you please post your Dockerfile?

**Comment by 1011382654** [comment]
> 

I try to run outside docker ERROR:Running as root without --no-sandbox is not
I found the answer in Google. Where should I add it.
puppetter.launch({args: ['--no-sandbox', '--disable-setuid-sandbox']})
quote:https://techozu.com/fix-running-as-root-without-no-sandbox-error-in-puppeteer/

**Comment by drudge** [comment]
Upgrade to v0.6.2 and restart n8n. After restart, you'll see a new option **Launch Arguments**. You can add both of them there.

**Comment by 1011382654** [comment]
> Upgrade to v0.6.2 and restart n8n. After restart, you'll see a new option **Launch Arguments**. You can add both of them there.

Thanks, all right

**Comment by drudge** [comment]
> > Upgrade to v0.6.2 and restart n8n. After restart, you'll see a new option **Launch Arguments**. You can add both of them there.
> 
> Thanks, all right

Did that solve your issue?

**Comment by 1011382654** [comment]
> > > 升级到 v0.6.2 并重启 n8n。重新启动后，您将看到一个新选项**Launch Arguments**。你可以在那里添加它们。
> > 
> > 
> > 谢谢，好的
> 
> 这解决了你的问题吗？

Yes

**Comment by drudge** [comment]
Glad to hear it. Have a great day!

**Comment by minhlucvan** [comment]
Hi guys, I’ve been looking for a dynamic scraping and browser automation integrated with n8n for a while. After trying many ways I found [browserless](https://www.browserless.io/) which perfectly fits my personal case so I decided to implement a community node called n8n-node-browserless which enables the powerful API of browser automation into n8n workflow.

NPM package [n8n-nodes-browserless - npm](https://www.npmjs.com/package/n8n-nodes-browserless)

GitHub [GitHub - minhlucvan/n8n-nodes-browserless: n8n node to interact with browserless instance](https://github.com/minhlucvan/n8n-nodes-browserless)

project is in the early stage so might need contributions from those who are interested in this topic.
